config.toml

```
@@ -117,6 +117,13 @@ languageName = "বাংলা(Bengali)"
 contentDir = "content/bn"
 weight = 8
 
+[languages.fr]
+title = "Glossaire Cloud Native"
+description = "Le projet de Glossaire CNCF Cloud Natif a pour objectif de référencer les termes courants utilisés dans les différentes applications cloud native."
+languageName ="Français (French)"
```

```suggestion
languageName ="Français(French)"
```

For a consistency with other languages. (language selection UI is not long)

---

Hi @fydrah
Note: I will add a new commit to apply my suggestion. :)
Actually, this is an intentional commit to initiate a Hugo build by Netlify. (we applied Netlify to dev-fr recently)
Netlify will provide a live preview site. :)